kanji/07006.svg

```diff
@@ -0,0 +1,241 @@
+       style="fill:none;stroke:#000000;stroke-width:3;stroke-linecap:round;stroke-linejoin:round"
+       inkscape:label="kvg:07006-g1"><path
+         id="kvg:06df1-s1"
+         kvg:type="㇔"
+         d="m 15.88,15.75 c 3.31,1.47 7.54,5.05 8.37,7.34"
+         inkscape:label="kvg:07006-s1" /><path
+         id="kvg:06df1-s2"
+         kvg:type="㇔"
+         d="m 12,39.75 c 3.79,1.15 8.8,4.97 9.75,6.75"
+         inkscape:label="kvg:07006-s2" /><path
+         id="kvg:06df1-s3"
+         kvg:type="㇀"
+         d="m 12,90.46 c 2.38,0.17 3.31,-0.89 4.25,-2.5 2.75,-4.7 5.5,-10.45 8,-16.45"
+         inkscape:label="kvg:07006-s3" /></g></g><g
+     id="g326"
+     inkscape:label="kvg:StrokeNumbers_07006"><text
+       id="text25"
+       style="font-size:8px;fill:#808080"
+       x="8.7281008"
+       y="12.767094">1</text><text
+       id="text27"
+       style="font-size:8px;fill:#808080"
+       x="7.3797579"
+       y="38.54599">2</text><text
+       id="text29"
+       style="font-size:8px;fill:#808080"
+       x="6.2495298"
+       y="95.752159">3</text><text
+       id="text31"
+       style="font-size:8px;fill:#808080"
+       x="34.17189"
+       y="16.299828">4</text><text
+       id="text33"
+       style="font-size:8px;fill:#808080"
+       x="54.67968"
+       y="8.9472523">5</text><text
+       id="text35"
+       style="font-size:8px;fill:#808080"
+       x="39.040806"
+       y="28.614492">6</text><text
+       id="text37"
+       style="font-size:8px;fill:#808080"
+       x="26.439774"
+       y="41.917286">7</text><text
+       id="text39"
+       style="font-size:8px;fill:#808080"
+       x="34.494438"
+       y="34.738899">8</text><text
+       id="text41"
+       style="font-size:8px;fill:#808080"
+       x="47.34729"
+       y="44.213673">9</text><text
+       id="text43"
+       style="font-size:8px;fill:#808080"
+       x="61.04509"
+       y="42.551079">10</text><text
+       id="text45"
+       style="font-size:8px;fill:#808080"
+       x="35.697693"
+       y="48.611263">11</text><text
+       id="text47"
+       style="font-size:8px;fill:#808080"
+       x="34.791313"
+       y="60.478222">12</text><text
+       id="text49"
+       style="font-size:8px;fill:#808080"
+       x="45.74115"
+       y="54.741714">13</text><text
+       id="text51"
+       style="font-size:8px;fill:#808080"
+       x="49.519024"
+       y="63.612625">14</text><text
+       id="text53"
+       style="font-size:8px;fill:#808080"
+       x="49.444576"
+       y="72.349785">15</text><text
+       id="text53-3"
+       style="font-size:8px;fill:#808080"
+       x="49.269089"
+       y="82.246788">16</text><text
+       id="text53-3-6"
+       style="font-size:8px;fill:#808080"
+       x="39.884808"
+       y="91.536293">17</text><text
+       id="text53-3-6-7"
+       style="font-size:8.80355px;fill:#808080;stroke-width:1.10044"
+       x="53.566883"
+       y="99.935242"
+       transform="scale(1.1004438,0.90872429)">18</text></g></svg>
```

Comment:
Can you remove these transform commands? They aren't used anywhere else in KanjiVG.